core/src/test/scala/ai/lum/odinson/events/EventSpec.scala

```diff
@@ -0,0 +1,58 @@
+package ai.lum.odinson.events
+
+import ai.lum.odinson.{BaseSpec, OdinsonMatch, EventMatch}
+
+import org.scalatest._
+
+class EventSpec extends BaseSpec {
+  def testEventTrigger(m: OdinsonMatch, start: Int, end: Int): Unit = {
+    m shouldBe an [EventMatch]
+    val em = m.asInstanceOf[EventMatch]
+    val trigger = em.trigger
+    trigger.start shouldEqual start
+    trigger.end shouldEqual end
+  }
+
+  def testEventArguments(m: OdinsonMatch, desiredArgs: Seq[Argument]): Unit = {
+    // extract match arguments from the mathing objects
+    val matchArgs = for(nc <- m.namedCaptures) 
+      yield Argument(nc.name, nc.capturedMatch.start, nc.capturedMatch.end)
+      // all desired args should be there, in the right number
+      val groupedMatched = matchArgs.groupBy(_.name)
+      val groupedDesired = desiredArgs.groupBy(_.name)
+      //
+      for ((desiredRole, desired) <- groupedDesired) {
+        // there should be arg(s) of the desired label
+        groupedMatched.keySet should contain (desiredRole)
+        // should have the same number of arguments of that label
+        val matchedForThisRole = groupedMatched(desiredRole)
+        desired should have size matchedForThisRole.size
+        for (d <- desired) {
+          matchedForThisRole should contain (d)
+        }
+        // there shouldn't be any found arguments that we didn't want
+        val unwantedArgs = groupedMatched.keySet.diff(groupedDesired.keySet)
+        unwantedArgs shouldBe empty
+      }
+  }
+
+  def createArgument(name: String, start: Int, end: Int): Argument = Argument(name, start, end)
+  
+  def getJsonDocument(id: String): String = {
+    val json = Map(
+      "1" -> """{"id":"48fb577b-f5ba-4e16-864f-f8ba20ba9cfa","metadata":[],"sentences":[{"numTokens":8,"fields":[{"$type":"ai.lum.odinson.TokensField","name":"raw","tokens":["The","consumption","of","gummy","bears","and","donuts","."],"store":true},{"$type":"ai.lum.odinson.TokensField","name":"word","tokens":["The","consumption","of","gummy","bears","and","donuts","."]},{"$type":"ai.lum.odinson.TokensField","name":"tag","tokens":["DT","NN","IN","NN","NNS","CC","NNS","."]},{"$type":"ai.lum.odinson.TokensField","name":"lemma","tokens":["the","consumption","of","gummy","bear","and","donut","."]},{"$type":"ai.lum.odinson.TokensField","name":"entity","tokens":["O","O","O","B-dessert","I-dessert","O","B-dessert","O"]},{"$type":"ai.lum.odinson.TokensField","name":"chunk","tokens":["B-NP","I-NP","B-PP","B-NP","I-NP","O","B-NP","O"]},{"$type":"ai.lum.odinson.GraphField","name":"dependencies","edges":[[1,0,"det"],[1,4,"nmod_of"],[1,7,"punct"],[4,2,"case"],[4,3,"compound"],[4,5,"cc"],[4,6,"conj"]],"roots":[1]}]}]}""",
```

Comment:
Having all the docs in one place is great, but can you give them names that make it easier to know which one we're grabbing? Maybe add as a comment a mapping from name to contents, i.e., the actual sentences in the doc.

Reply:
Do you think using the id is enough?

I was planning to have a TAG svg of all documents used for test somewhere in a readme.
Meanwhile the only thing I could think would be informative would be the document id.jk